blocks/footer/footer.css

```diff
@@ -1,16 +1,325 @@
+      text-decoration: none !important;
+      font-weight: normal;
+      transition: all 0.3s ease-in-out;
+      &:hover { color: var(--yellow) }
+    }
+
+    h4 {
+      font-size: 1.1em;
+      font-family: var(--heading-font-family);
+      font-weight: 400;
+      margin: 16px 0;
+      a:hover { color: #fff }
+    }
+
+    ul {
+      list-style: none;
+      width: 100%;
+      li {
+        display: inline-block;
+        width: 49%;
+        line-height: 1.7em;
+        font-size: .8em;
+        white-space: nowrap;
+      }
+    }
+
+    .btn {
+      display: inline-block;
+      font-size: .8em;
+      line-height: 1.5;
+      text-align: center;
+      white-space: nowrap;
+      vertical-align: middle;
+      touch-action: manipulation;
+      cursor: pointer;
+      user-select: none;
+      border: 1px solid transparent;
+      border-radius: 3px;
+      background-image: none;
+      border-color: #1b4b85;
+      color: #fff;
+      padding: 4px 0;
+      transition: all 0.3s ease-in-out;
+      font-weight: normal;
+      &:hover {
+        color: var(--yellow);
+        border-color: var(--yellow);
+      }
+    }
+
+    > div {
+      display: flex;
+      flex-direction: row;
+      max-width: 1200px;
+      margin: auto;
+    }
+  
+    .col-1, .col-2, .col-3 {
+      width: 30%;
+      height: 260px;
+      padding: 0 50px;
+      position: relative;
+      p {
+        width: 80%;
+        position: absolute;
+        bottom: 0;
+      }
+    }
+
+    .col-1 {
+      .btn {
+        width: 40%;
+        &:nth-child(1) {
+          margin-right: 8%;
+        }
+      }
+    }
+
+    .col-2 {
+      /* vertical lines */
+      &::before, &::after {
+        background: rgb(255 255 255);
+        background: linear-gradient(180deg, rgba(255 255 255 / 0%) 0%, rgba(27 75 133 / 100%) 50%, rgba(255 255 255 / 0%) 100%);
+        width: 1px;
+        content: '';
+        display: block;
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        min-height: 70px;
+      }
+      &::after {
+        left: auto;
+        right: 0;
+      }
+
+      form.subscribe {
+        position: absolute;
+        bottom: 0;
+        width: 75%;
+        small {
+          display: block;
+          width: inherit;
+          text-align: center;
+          padding: 4px 0;
+          font-size: .7em;
+          white-space: nowrap;
+        }
+        input {
+          display: inline-block;
+          box-sizing: border-box;
+          height: 30px;
+          width: calc(50% - 2px);
+          font: inherit;
+          font-family: inherit;
+          padding: 6px 12px;
+          margin: 1px;
+          font-size: .9em;
+          line-height: 1.5;
+          color: #555;
+          background: #fff;
+          border-radius: 4px;
+          border: none;
+          vertical-align: middle;
+          outline: none;
+          transition: all ease-in-out .15s;
+          &::placeholder { color: #555 }
```

Comment:
The hubblehomes.com text color for placeholders looks like #999.  
![image](https://github.com/aemsites/hubblehomes-com/assets/6395150/3ef3a496-521d-4a23-80c7-42af722e0e9c)

Reply:
Eagle eyes! :)